content/lessons/chapter-2/outro-1.tsx

```diff
@@ -7,7 +7,7 @@ export const metadata = {
   title: 'chapter_two.outro_one.title',
   image: '/assets/images/chapter-2-finish.jpg',
   theme: 'bg-transparent/20',
-  secondaryTheme: 'bg-[#362042]',
+  secondaryTheme: 'bg-[#3c5c6e]',
```

Comment:
Looks like you accidentally made the color change for chapter-2

Reply:
Oops, sorry! 😅
Let's fix it! ✨

Reply:
Fixed it in 625aa12a29bad54faa7d722a979c0d427abd53fb
Thanks for catching it!